### ISSUE TITLE
Canon EOS R50 Noise Profile

### ISSUE THREAD

**Issue by laemplein** [darktable-org/darktable#14574]
Hey there, I have created a noise profile for Canon EOS R50, I hope this is helpfull.

[darktable-noiseprofile-20230524.tar.gz](https://github.com/darktable-org/darktable/files/11559397/darktable-noiseprofile-20230524.tar.gz)

**Comment by kmilos** [comment]
Fyi, there is no base support for the R50 yet: https://github.com/darktable-org/darktable/issues/14549#issuecomment-1558554926

**Comment by laemplein** [comment]
Oh hey, thank you for the heads up! Interesting, I was able to open and edit the RAW and cRAW files... Well I will then wait for it to be officially supported and create the noise profile again after that. Just in case something changes with interpreting these files.

**Comment by kmilos** [comment]
> Interesting, I was able to open and edit the RAW and cRAW files...

Yeah, the underlying LibRaw library used for reading CR3s doesn't necessarily give up (or warn you) for new/unknown cameras, but gives you a best string match - in this case you're probably getting the input profile of an R5...

**Comment by ralfbrown** [comment]
Done.  But if there isn't actually proper base support for the R50 yet (rather than your surmised fuzzy match by LibRaw), shouldn't this *also* be tagged "camera support"?

**Comment by mattipulkkinen** [comment]
Does the new [Libraw release](https://github.com/LibRaw/LibRaw/releases) mean that Darktable now has, or will soon have, support for the R50?

**Comment by kmilos** [comment]
> Does the new [Libraw release](https://github.com/LibRaw/LibRaw/releases) mean that Darktable now has, or will soon have, support for the R50?

Unfortunately not yet - it is only a minor bugfix release, not a "major" public snapshot w/ added camera support: https://www.libraw.org/#updatepolicy
